src/main/java/dk/cngroup/lentils/service/CypherStatus.java

```diff
@@ -0,0 +1,18 @@
+package dk.cngroup.lentils.service;
+
+public enum CypherStatus {
```

Comment:
Toto asi nebude spravne. Jednak pokud nekdo skipne sifru, tak je to za 0b. Dal chybi mapovani situace, kdy si vezmou hint a pak to vzdaji a preskoci (neco jako SKIPPED_WITH_HINT). (Tj. ve vysledku budou -5 bodu).